docs/quickstart/installing.md

```diff
@@ -38,37 +38,10 @@ You can try lakeFS:
 
 ## Modifying the local deployment
 
-To modify the local deployment, for example, in order to use your local lakeFS against S3 storage (as opposed to the local storage), you can either:
-1. Pass or expose the variables using this syntax:
-
-   ```bash
-   curl https://compose.lakefs.io | LAKEFS_BLOCKSTORE_TYPE=s3 AWS_ACCESS_KEY_ID=YourAccessKeyValue AWS_SECRET_ACCESS_KEY=YourSecretKeyValue docker-compose -f - up
-   ```
-2. Download the configuration file https://compose.lakefs.io, modify it and then run the container with the modified copy:
-
-   ```bash
-   docker-compose -f modified-docker-compose.yml up
-   ```
-
-   For example, to run against S3 instead of local storage, change:
-   ```bash
-   ...
-   - LAKEFS_BLOCKSTORE_TYPE=${LAKEFS_BLOCKSTORE_TYPE:-local}
-   ...
-   - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID:-}
-   - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_SECRET_KEY=${AWS_SECRET_ACCESS_KEY:-}
-   ...
-   ```
-   To:
+To modify the local deployment, for example, in order to use your local lakeFS against S3 storage (as opposed to the local storage), run the command with local parameters:
    ```bash
-   ...
-   - LAKEFS_BLOCKSTORE_TYPE=s3
-   ...
-   - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_KEY_ID=###
-   - LAKEFS_BLOCKSTORE_S3_CREDENTIALS_ACCESS_SECRET_KEY=###
-   ...
+docker run --pull always -p 8000:8000 -e LAKEFS_BLOCKSTORE_TYPE='s3' -e AWS_ACCESS_KEY_ID='XXX' -e AWS_SECRET_ACCESS_KEY='YYY'  treeverse/lakefs run --local-settings
    ```
```

Comment:
I would add a note about that that some of the metadata is being stored locally in lakeFS container, therefore the `--local-settings`.  Users should avoid using that flag for real usages.